.github/PULL_REQUEST_TEMPLATE/bug_PR.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Bug PR
+about: Submit a PR for a bug
+title: 'BUG: '
+labels: bug
+assignees: ''
+
+---
```

Comment:
```suggestion
```
We discussed, that we are only going to do one PR template regardless of the label.
If the correct prefix was chosen for the PR name will be checked by a GitHub action.
Also the label can be applied automatically with a Github action.
So with only one template this box feels unnecessary.